testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -2913,6 +2913,12 @@ void checkPeriodPredicate(Checker checker) {
     sql(sql).ok(expected);
   }
 
+  @Test void testDateFunction() {
```

Comment:
this function requires the babel parser

Reply:
@mihaibudiu Hi,It is already in babel parser but has not been ported to SqlParserTest, probably because parser.jj does not support the parsing of certain datefunctions.